app/assets/javascripts/utility.js

```diff
@@ -1,3 +1,5 @@
+/* exported utility */
```

Comment:
It's supposed to fix the bug that said utility was assigned but never used, by saying this it should see that it is used elsewhere.